index.html

```diff
@@ -9,6 +9,9 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Hello World</h1> 
+  <h1 class="titulo-libro"><strong>Título del libro</strong> <p class= "titulo-genero"> (género)</p></h1> <br>
```

Comment:
How about identing the element like this:
```suggestion
<h1 class="titulo-libro">
  <strong>Título del libro</strong>
  <p class= "titulo-genero"> (género)</p>
</h1>
```